src/Message/CatalogMessage.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Netflie\WhatsAppCloudApi\Message;
+
+final class CatalogMessage extends Message
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected string $type = 'catalog_message';
+
+    private string $body;
+
+    private string $footer;
+
+    private string $thumbnail;
```

Comment:
same

Reply:
Unbundled and fixed in PR https://github.com/netflie/whatsapp-cloud-api/pull/191